extension/src/background/helpers/dataStorage.ts

```diff
@@ -105,7 +105,7 @@ export const normalizeMigratedData = async () => {
 
 // This migration adds a friendbotUrl to testnet and futurenet network details
 export const migrateFriendBotUrlNetworkDetails = async () => {
-  const localStore = dataStorageAccess(browserLocalStorage);
+  const localStore = exports.dataStorageAccess(browserLocalStorage);
```

Comment:
we have to assign these to "exports" in order to be able to properly mock them with jest. This should be equivalent to local reference.

Reply:
Interesting - I've never had to do this. Can you point me to some documentation on this?